### PR TITLE
Exit installation if user cancels it when prompted for password

### PR DESCRIPTION
**Pull request by sonicaj**

### CODE REVIEW

**Review by william-gr — COMMENTED** [review]
Hmm why, in theory cancel password should work just fine. UI has menas to set it on first use.

**Comment by william-gr** [comment]
My bad on the ticket, i didnt realize it at first.

**Comment by sonicaj** [comment]
Screenshot 2020-09-22 at 1 21 13 PM
@william-gr this is the updated dialog, please let me know if we would require changes to the layout.

File: usr/sbin/truenas-install [hunk 1]
```diff
@@ -624,23 +624,28 @@ __EOF__
 
     while true; do
 	dialog --insecure \
+	    --extra-button --extra-label "Donot set password" \
```
Comment (william-gr) [inline usr/sbin/truenas-install:627]: Do not, missing space?

Or just Dont

**Review by william-gr — CHANGES_REQUESTED** [review]

**Review by william-gr — APPROVED** [review]

**Pull request merged**